api/proto/admin.proto

```
@@ -0,0 +1,265 @@
+  rpc GetRoute(GetRouteRequest) returns (AdminRoute);
+  rpc AddRoute(AddRouteRequest) returns (AdminRoute);
+  rpc UpdateRoute(UpdateRouteRequest) returns (AdminRoute);
+  rpc RemoveRoute(RemoveRouteRequest) returns (RemoveRouteResponse);
+
+  // Secrets
+  rpc ListSecrets(ListSecretsRequest) returns (ListSecretsResponse);
+  rpc SetSecrets(SetSecretsRequest) returns (SetSecretsResponse);
+  rpc DeleteSecret(DeleteSecretRequest) returns (DeleteSecretResponse);
+
+  // Logs - server streaming for real-time logs
+  rpc GetProcessLogs(GetProcessLogsRequest) returns (stream LogEntry);
```

Buf (1.64.0)

[error] "gordon.LogEntry" is used as the request or response type for multiple RPCs.

(RPC_REQUEST_RESPONSE_UNIQUE)